analytics-api/src/analytics_api/resources/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Exposes all of the resource endpoints mounted in Flask-Blueprint style.
+
+Uses restplus namespaces to mount individual api endpoints into the service.
+
+All services have 2 defaults sets of endpoints:
+ - ops
+ - meta
+That are used to expose operational health information about the service, and meta information.
+"""
+
+from flask import Blueprint
+
+from .apihelper import Api
+from .engagement import API as ENGAGEMENT_API
+__all__ = ('API_BLUEPRINT',)
+
+URL_PREFIX = '/api/'
```

Comment:
i think we are dropping the url prefix, i'm not sure yet

Reply:
lets keep it

Reply:
ok